### ISSUE TITLE
Long-term tracking

### ISSUE THREAD

**Issue by TinajaLabs** [owntracks/owntracks#17]
so will ask here...  My main feature request would be to have a trail showing my path while on vacation.  I only see in the android app where I am, but not where I've been.

How would I see that?

Thanks,
Chris.

**Comment by jpmens** [comment]
Use our [Recorder](https://github.com/owntracks/recorder) for this.

**Comment by TinajaLabs** [comment]
Thank you sir.  I don't have the stand-alone version installed on my home server yet, but will do so soon.

BTW, I've tracked your work with MQTT, pub/sub, mosquito, etc. and I have a home automation system built around a collection of tech including raspis, node-red, graphite/grafana, xbee and esp8266 radios, old X10, etc.  

For the last 5 years I've had a booth at the SF Bay Area Maker Faire showing how I have my fun.  My most recent addition was installing Rocket.chat and hubot on a raspi and configuring it to communicate with my node-red controller so I can chat commands with my home.

Last fall I set up a puppet server to manage the deployment of my raspberry pis in a repeatable fashion.  Now I can go from a fresh raspbian install to, for example, a fully configured node-red with all my configured nodes in minutes instead of an hour or so.

Part of my day job is managing a monitoring system build on Icinga, so my hobby and my work cross over a lot.  I love what I do and your articles have been an inspiration.  Thanks.
